DESCRIPTION

```diff
@@ -27,9 +27,9 @@ Authors@R: c(
 Description: Bayesian dynamic borrowing is an approach to incorporating external
    data to supplement a randomized, controlled trial analysis in which
    external data are incorporated in a dynamic way (e.g., based on similarity
-   of outcomes); see Viele 2013 \doi{10.1002/pst.1589} for an overview.
+   of outcomes); see Viele 2013 <doi:10.1002/pst.1589> for an overview.
    This package implements the hierarchical commensurate prior approach to dynamic borrowing
-   as described in Hobbes 2011 \doi{10.1111/j.1541-0420.2011.01564.x}.
+   as described in Hobbes 2011 <doi:10.1111/j.1541-0420.2011.01564.x>.
    There are three main functionalities. First, 'psborrow2' provides a user-friendly 
    interface for applying dynamic borrowing on the study results handles the Markov Chain
    Monte Carlo sampling on behalf of the user. Second, 'psborrow2' provides a 
```

Comment:
From the CRAN policies

> Packages on which a CRAN package depends should be available from a mainstream repository: if any mentioned in ‘Suggests’ or ‘Enhances’ fields are not from such a repository, where to obtain them at a repository should be specified in an ‘Additional_repositories’ field of the DESCRIPTION file (as a comma-separated list of repository URLs) or for other means of access, described in the ‘Description’ field.

To me this reads that `Additional_repositories` is sufficient. My guess is they also want something about installing cmdstanr in `Description`

This package is designed to use the sampling functions from 'cmdstanr' which can be installed from <https://mc-stan.org/r-packages/>